### DISCUSSION TITLE
Is it possible to link to a different workload item?

### DISCUSSION THREAD

**Discussion by q-benwillis** [microsoft/Microsoft-Fabric-workload-development-sample#144] [Help] [answered]
We would like to add a link in our workload item that links out to a semantic model still within Fabric. Is this possible within the current framework?

The iFrame doesn't have the [allow-top-navigation](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/iframe#allow-top-navigation) or [allow-popups](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/iframe#allow-popups) sandbox controls which I believe means we can't use a regular hyperlink.

The docs say that the `workloadName` in the [OpenPageConfig](https://learn.microsoft.com/en-us/javascript/api/@ms-fabric/workload-client/openpageconfig?view=fabric-extensibility-js-latest) is ignored and the open page API can only be used to open pages in the current workload.

Is my understanding correct? Is there another way we can achieve this?

We're looking for something similar to the open new notebook button you have in the lakehouse workload:
![image](https://github.com/user-attachments/assets/e50815a5-d46d-44bd-8eb8-f9162ec8e6f9)

**Comment by anitamayorov** [comment, marked as answer]
Hi @q-benwillis , this experience is currently not supported, no ETA. Regarding iframe - yes, it is sandboxed and with 'allow-same-origin' and 'allow-scripts' attributes only.
